languages/zh-tw/constants.genie

```diff
@@ -0,0 +1,134 @@
+    duration:constant_Measure_ms '之前' => new Ast.Value.Date(null, '-', duration);
+    duration:constant_Measure_ms pt:constant_date_point '以後' => new Ast.Value.Date(pt.value, '+', duration);
+    duration:constant_Measure_ms pt:constant_date_point '之後' => new Ast.Value.Date(pt.value, '+', duration);
+    duration:constant_Measure_ms pt:constant_date_point '以前' => new Ast.Value.Date(pt.value, '-', duration);
+    duration:constant_Measure_ms pt:constant_date_point '之前' => new Ast.Value.Date(pt.value, '-', duration);
+    }
+}
+
+constant_Measure_byte = {
+    !turking num:constant_Number 'byte' => C.addUnit('byte', num);
+    num:constant_Number 'kb' => C.addUnit('KB', num);
+    num:constant_Number 'mb' => C.addUnit('MB', num);
+    num:constant_Number 'gb' => C.addUnit('GB', num);
+    num:constant_Number 'tb' => C.addUnit('TB', num);
+    num:constant_Number 'kilobytes' => C.addUnit('KB', num);
+    num:constant_Number 'megabytes' => C.addUnit('MB', num);
+    num:constant_Number 'gigabytes' => C.addUnit('GB', num);
+    num:constant_Number 'terabytes' => C.addUnit('TB', num);
+}
+
+constant_Measure_kg = {
+    !turking num:constant_Number '公克' => C.addUnit('g', num);
+    num:constant_Number '公斤' => C.addUnit('kg', num);
+    num:constant_Number '磅'   => C.addUnit('lb', num);
+    num:constant_Number '盎司' => C.addUnit('oz', num);
+}
+
+constant_Measure_C = {
+    !turking {
+        num:constant_Number 'c' => C.addUnit('C', num);
+        num:constant_Number '攝氏' => C.addUnit('C', num);
+    }
+    num:constant_Number 'f' => C.addUnit('F', num);
+    num:constant_Number '華氏' => C.addUnit('F', num);
+    num:constant_Number '度' => C.addUnit('TB', num);
+}
+
+constant_Measure_m = {
+    !turking num:constant_Number '米' => C.addUnit('m', num);
+    num:constant_Number '公尺' => C.addUnit('m', num);
+    num:constant_Number '公里' => C.addUnit('km', num);
+    num:constant_Number '公里' => C.addUnit('km', num);
+    num:constant_Number '英里' => C.addUnit('mi', num);
+    num:constant_Number '英里' => C.addUnit('mi', num);
+    num:constant_Number '英尺' => C.addUnit('ft', num);
+    num:constant_Number '英寸' => C.addUnit('in', num);
+    num:constant_Number '英寸' => C.addUnit('in', num);
+    num:constant_Number '公分' => C.addUnit('cm', num);
+    num:constant_Number '厘米' => C.addUnit('cm', num);
+}
+}
+
+constant_Measure_mps = {
+    num:constant_Number 'mph' => C.addUnit('mph', num);
+    !turking num:constant_Number 'm/s' => C.addUnit('mps', num);
+    num:constant_Number 'kph' => C.addUnit('kmph', num);
+    num:constant_Number 'miles per hour' => C.addUnit('mph', num);
+    num:constant_Number 'kilometers per hour' => C.addUnit('kmph', num);
+    num:constant_Number 'km/h' => C.addUnit('kmph', num);
+}
+
+constant_Location = {
+    '這裡'                    => new Ast.Value.Location(Ast.Location.Relative('current_location'));
+    !turking '我 在 哪裡' => new Ast.Value.Location(Ast.Location.Relative('current_location'));
```

Comment:
This is a question form, correct? The English template `where I am now` was meant for a relative clause used a parameter, as in `call an uber from where I am now to ...` or `find me a restaurant around where I am now`

Reply:
Yeah that is question form. 
Misunderstanding this, fix.